EpiAware/src/EpiAwareBase/EpiAwareBase.jl

```diff
@@ -6,9 +6,17 @@ Module for defining abstract epidemiological types.
 
 using DocStringExtensions
 
-export AbstractModel, AbstractEpiModel, AbstractLatentModel,
-       AbstractObservationModel, AbstractEpiAwareProblem, generate_latent,
-       generate_latent_infs, generate_observations
+#Export models
+export AbstractModel, AbstractEpiModel, AbstractLatentModel, AbstractObservationModel
+
+#Export problems
+export AbstractEpiAwareProblem
+
+#Export inference methods
+export AbstractEpiAwareMethod, AbstractEpiAwareOptMethod, AbstractEpiAwareSamplingMethod
```

Comment:
Are we happy that he naming is consistent w/ these? The others don't have EpiAware nested within their name.

Reply:
I'm not 100% happy, and open to a suggestion here.

Maybe, `AbstractEpiAwareProblem` but others are `AbstractMethod`, `AbstractEpiAwareSamplingMethod` etc etc?

Reply:
Yeah so `AbstractEpiAwareMethod` and everything else follows the other naming scheme?

Reply:
E.g. 

`AbstractOptMethod <: AbstractEpiAwareMethod`

etc?

Reply:
or just do epi for all of them?

Reply:
Lets do that.